### DISCUSSION TITLE
Year In Review stats

### DISCUSSION THREAD

**Discussion by thejdlowe** [RetroAchievements/RAWeb#2116] [Ideas]
I realize this is far too late to try to implement in any meaningful way, but it'd be awesome to have something similar to Steam/Nintendo/etc. where we can see all of our stats; a breakdown from the past year of how many games per how many consoles played, and how many achievements. How many mastered games! What was the achievement you got that was worth the most points? Et cetera!

**Comment by hb-robo** [comment]
I admit to being a sucker for "Wrapped" style content, so I'm very into this. 2024 maybe!

**Comment by Jamiras** [comment]
Done in #2905.
